src/sentry/web/frontend/debug/mail.py

```diff
@@ -456,7 +464,29 @@ def build_usage_summary():
             int(random.weibullvariate(5, 1) * random.paretovariate(0.2)),
         )
 
-    def build_report():
+    def build_calendar_data(project):
```

Comment:
Oops, forgot to add this to the switch configuration during the integration test. I'll do that — good catch.